recipes/deblur/meta.yaml

```diff
@@ -25,8 +25,8 @@ requirements:
     - setuptools
     - python
     - scikit-bio
-    - vsearch
-    - mafft
+    - vsearch ==2.0.3
```

Comment:
should the version spec for vsearch and mafft also be set in the `build` section?